docker/bookkeeper/entrypoint.sh

```diff
@@ -74,6 +75,27 @@ create_bookie_dirs() {
   done
 }
 
+# Create a Bookie ID if this is a newly added bookkeeper pod
+# or read the Bookie ID if it already exists
+set_bookieid() {
+  IFS=',' read -ra directories <<< $1
+  BK_ID_DIR="${directories[0]}"
```

Comment:
Here you are relying in the fact that the `directories` list will always have the list of Bookkeeper directories sorted in the same order. Do you know if that assumption will hold irrespective of the environment?

Reply:
The webhook in the bookkeeper operator code ensures that the contents of the `directories` is not modifiable, hence it is guaranteed to have the directories sorted in the same order at all times.

Reply:
This is worth a comment saying that there is an assumption on the order of directories. Just so that I understand, where is it getting `directories`?

Reply:
@fpj it gets this value from the `options` field in the bookkeeper manifest